src/create/unknown.js

```diff
@@ -30,20 +32,23 @@ function createUnknown(
     case 'block':
         return Block.create({
             type: tagName,
+            key,
             nodes: createTextNodes(children),
-            ...attributes
+            ...otherAttributes
```

Comment:
Wasn't `...attributes` already passing `key`?

Reply:
it was also passing "kind", which we don't want